pkg/executor/executor.go

```diff
@@ -366,18 +367,20 @@ func (e *ShowDDLExec) Next(ctx context.Context, req *chunk.Chunk) error {
 		return nil
 	}
 
-	ddlJobs := ""
 	query := ""
 	l := len(e.ddlInfo.Jobs)
 	for i, job := range e.ddlInfo.Jobs {
-		ddlJobs += job.String()
 		query += job.Query
 		if i != l-1 {
-			ddlJobs += "\n"
 			query += "\n"
 		}
 	}
 
+	ddlJobs, err := json.Marshal(e.ddlInfo.Jobs)
```

Comment:
It would print many unnecessary fields.

Reply:
I think it's fine to have all the fields, more fields means easier debugging.